cmd/fynedesk/main.go

```diff
@@ -1,12 +1,27 @@
 package main
 
 import (
+	"os"
+
+	"fyne.io/fyne"
 	"fyne.io/fyne/app"
 )
 
+func startSettingsListener(app fyne.App, listener chan fyne.Settings) {
+	for {
+		_ = <-listener
+		if os.Getenv("FYNE_DESK_RUNNER") != "" {
```

Comment:
The issue here - and we can revisit this over the next few days - is how to do the refresh when it will require access to multiple packages that don't have access to each other.

Reply:
This listener could be put in a non-main package